addons/mail/tests/test_notify_security_update.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.addons.mail.tests.common import MailCommon
+from odoo.tests import users
+
+
+class TestNotifySecurityUpdate(MailCommon):
+    @users('employee')
+    def test_security_update_email(self):
+        """ User should be notified on both email addresses when the email changes """
+        self.env.user.write({'email': 'new@example.com'})
```

Comment:
I suggest to use mock tools, find mail.mail and check their content with available asserts and tools :)